### PR TITLE
Integrate drizzle with SQLite

### PR DESCRIPTION
**Pull request by lpbonomi**
Related to #143

### CODE REVIEW

**Comment by luwes** [comment]
thanks! this looks great already! will add some notes

File: src/database/sqlite/adapter.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      'error',
+    ]}).notNull(),
+    originalFilePath: text('original_file_path').notNull(),
+    provider: text('provider', {enum: [ 'mux',  'vercel-blob', 'backblaze', 'amazon-s3']}).notNull(),
+    providerMetadata: text('provider_metadata', {mode: 'json'}).notNull(),
+    poster: text('poster'),
+    blurDataURL: text('blur_data_url'),
+    size: integer('size'),
+    error: text('error'),
+    externalIds: text('external_ids', {mode: 'json'}),
```
Comment (luwes) [inline src/database/sqlite/adapter.ts:36]: `externalIds` can be removed here. these are stored in providerMetadata in the new schema

Comment (lpbonomi) [reply src/database/sqlite/adapter.ts:36]: Great! I just put it in there because I saw some comments about backwards compatibility

**Comment by luwes** [comment]
thanks for contributing this so far! 

we will soon help on this feature as we plan to add support for an online upload flow (UGC). 
did you plan to add more to this or did you go with another solution?

**Comment by lpbonomi** [comment]
I'd like to keep helping with this. Any feedback or guidance is welcome :)

**Pull request closed without merging**